ansible/playbook.yml

```diff
@@ -3,4 +3,10 @@
   vars_files:
   - vars/main.yml
   roles:
+  - { role: geerlingguy.nginx }
+  - { role: geerlingguy.mysql }
+  - { role: geerlingguy.php }
+  - { role: geerlingguy.composer }
```

Comment:
these is a really big dependency list for such a simple nginx-php-mysql setup, I am not very comfortable with this, I do not want to depend on so many ansible roles that I would have to monitor, audit, ...

Reply:
Perhaps we could refactor it later? :D